src/server/active_tran_server.cpp

```diff
@@ -68,7 +68,48 @@ active_tran_server::get_remote_storage_config ()
 }
 
 void
-active_tran_server::receive_saved_lsa (page_server_conn_t::sequenced_payload &a_ip)
+active_tran_server::on_boot ()
```

Comment:
`on_boot` seems to serve no specific purpose anymore - except the assert, which is superfluous; can it be removed?

Reply:
I just left it to respect the writer, who may think a callback for booting is needed. I think we can remove it for now.

Reply:
done.